README.md

```diff
@@ -13,39 +14,28 @@
 ## Install
 
 ```sh
-yarn
+pnpm install
 ```
 
-## Build
-
 ```sh
-yarn build
+pnpm dev
 ```
 
-## Lint
+## Build
 
 ```sh
-yarn lint
+pnpm build
```

Comment:
Didn't build completely as shown here:
<img width="971" alt="image" src="https://github.com/subspace/auto-sdk-ts/assets/16472948/b54eb64a-ff68-4ead-a2fd-11ad64b678a2">


Here is the complete result:
```sh
❯ pnpm build

> @subspace/auto@ build /Users/abhi3700/F/coding/github_repos/subspace/auto-sdk-ts
> turbo run build

• Packages in scope: @auto/sdk, @subspace/auto-app, @subspace/eslint-config, @subspace/prettier-config, @subspace/tsconfig
• Running build in 5 packages
• Remote caching disabled
@auto/sdk:build: cache miss, executing c4c781c117f94b40
@auto/sdk:build: 
@auto/sdk:build: > @auto/sdk@0.0.1 build /Users/abhi3700/F/coding/github_repos/subspace/auto-sdk-ts/packages/sdk
@auto/sdk:build: > bob build
@auto/sdk:build: 
@auto/sdk:build: ℹ Building target commonjs
@auto/sdk:build: ℹ Cleaning up previous build at lib/commonjs
@auto/sdk:build: ℹ Compiling 13 files in src with babel
@auto/sdk:build: ✔ Wrote files to lib/commonjs
@auto/sdk:build: ℹ Building target module
@auto/sdk:build: ℹ Cleaning up previous build at lib/module
@auto/sdk:build: ℹ Compiling 13 files in src with babel
@auto/sdk:build: ✔ Wrote files to lib/module
@auto/sdk:build: ℹ Building target typescript
@auto/sdk:build: ℹ Cleaning up previous build at lib/typescript
@auto/sdk:build: ℹ Generating type definitions with tsc
@auto/sdk:build: ⚠ Failed to locate tsc in the workspace. Falling back to the binary found in PATH at /Users/abhi3700/F/coding/github_repos/subspace/auto-sdk-ts/node_modules/.bin/tsc. Consider adding typescript to your devDependencies or specifying the tsc option for the typescript target.
@auto/sdk:build: ✔ Wrote definition files to lib/typescript

 Tasks:    1 successful, 1 total
Cached:    0 cached, 1 total
  Time:    2.378s
```

Reply:
Fixed [here](https://github.com/subspace/auto-sdk-ts/pull/8/commits/d5ed326851517fa9ead193e532a02e4a30c3d809).